tests/scripts/exec_template.yml

```diff
@@ -12,3 +12,5 @@
       args:
         login: "{{ switch_login[hwsku_map[peer_hwsku]] }}"
       connection: switch
+      vars:
+        ansible_connection: switch
```

Comment:
@SavchukRomanLv , Can you add PR comment to describe why this change is needed?

Reply:
@yxieca we had similar issue with such errors:

TypeError: exec_command() got an unexpected keyword argument 'template'  fatal: [XX]: FAILED! => {  "msg": "Unexpected failure during module execution.",  "stdout": ""  }

 https://github.com/sonic-net/sonic-mgmt/issues/2387

Reply:
@SavchukRomanLv thanks for address this issue? Is this issue also caused by 'python3' which this PR's title is referencing fixing?

Both changes look good to me, I just don't think they should be in a single PR. Would you mind splitting them into 2 PRs?